_conferences/LitoralDroid-california-2017.md

```diff
@@ -0,0 +1,8 @@
+---
+name: "LitoralDroid"
+website: http://www.litoraldroid.com/
```

Comment:
This conference is strange. Was it like a talk ? I can't see a schedule, and the event seems to have finished already. I don't get it.